sdks/python/apache_beam/utils/profiler.py

```diff
@@ -134,10 +139,8 @@ def stop(self):
 
   def report_once(self):
     if not self._hpy:
-      logging.warning('guppy is not installed; MemoryReporter not available.')
       return
     report_start_time = time.time()
     heap_profile = self._hpy().heap()
-    logging.info('*** MemoryReport Heap:\n %s', heap_profile)
-    logging.info('*** MemoryReport took %.1f sec.',
-                 time.time() - report_start_time)
+    logging.info('*** MemoryReport Heap:\n %s\n MemoryReport took %.1f sec',
```

Comment:
Trying this out, Output looks like this

`...`
`<47 more rows. Type e.g. '_.more' to view.>`

Should we actually iterate over this to print the full picture? Otherwise it prints only the first 10 items.